cmd/nvidia-ctk/hook/hook.go

```diff
@@ -41,6 +41,9 @@ func (m hookCommand) build() *cli.Command {
 	hook := cli.Command{
 		Name:  "hook",
 		Usage: "A collection of hooks that may be injected into an OCI spec",
+		Action: func(ctx *cli.Context) error {
+			return commands.WarnOnUnsupportedSubcommand(m.logger, ctx)
+		},
 	}
 
 	hook.Subcommands = commands.New(m.logger)
```

Comment:
oh. Hm maybe "hook" is the command, and sub-commands are one level deeper.

Reply:
I hope my discussion above has cleared up the confusion here. We should make a pass of the existing code and update the comments / functions for consistency, but this has a low priority at present.